### PR TITLE
Use locally downloaded operator-sdk

### PR DESCRIPTION
**Pull request by clobrano**
This is necessary for post-update CI

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -226,6 +244,23 @@ OPM = $(shell which opm)
 endif
 endif
 
+.PHONY: operator-sdk
+operator-sdk: ## Download operator-sdk locally if necessary.
+	$(call operator-framework-tool, $(OPERATOR_SDK), $(OPERATOR_SDK_DIR),github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_$${OS}_$${ARCH})
+
+# operator-framework-tool will delete old package $2, then dowmload $3 to $1.
+define operator-framework-tool
```
Comment (razo7) [inline Makefile:252]: How about [`url-install-tool`]( https://github.com/medik8s/fence-agents-remediation/pull/3#discussion_r1081467799)?

File: Makefile [hunk 2]
```diff
@@ -226,6 +244,23 @@ OPM = $(shell which opm)
 endif
 endif
 
+.PHONY: operator-sdk
+operator-sdk: ## Download operator-sdk locally if necessary.
+	$(call operator-framework-tool, $(OPERATOR_SDK), $(OPERATOR_SDK_DIR),github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_$${OS}_$${ARCH})
+
+# operator-framework-tool will delete old package $2, then dowmload $3 to $1.
```
Comment (razo7) [inline Makefile:251]: typo `dowmload`-> `download`

File: Makefile (hunk 1 above)
Comment (razo7) [reply Makefile:252]: I agree, but `operator-framework-tool` name can be misleading as it is a tool from the operator-framework... I don't feel strongly to which direction, your choice.

Comment (razo7) [reply Makefile:252]: If we prefer to stick with **operator-framework** prefix then _operator-framework-tool_ or _operator-framework-install-tool_ makes more sense. I tend to go with just `operator-framework-tool`.

**Comment by mshitrit** [comment]
In SNR I haven't used `operator-framework-tool` just a simple `curl` :man_shrugging: (but I have no strong opinion either way)
Feel free to unhold.

/lgtm 
/hold

**Comment by clobrano** [comment]
> In SNR I haven't used `operator-framework-tool` just a simple `curl` man_shrugging (but I have no strong opinion either way) Feel free to unhold.
> 
> /lgtm /hold

Once it's set and working fine, I don't expect to change it much, so duplication is not a big deal, but I liked the approach of re-using a function :slightly_smiling_face: 

/unhold

**Pull request merged**